setup.cfg

```diff
@@ -16,6 +16,7 @@ classifiers =
 	Programming Language :: Python :: 3.9
 	Programming Language :: Python :: 3.10
 	Programming Language :: Python :: 3.11
+    Programming Language :: Python :: 3.12
```

Comment:
This line should use leading tab as the previous lines do.

Reply:
I don't know where might come the mistake, but it is actually correctly indented.

Reply:
updated.